### PR TITLE
mshr

### PR DESCRIPTION
**Pull request by minrk**
fenics mesh generation component, starting with 2016.2.0.

### CODE REVIEW

**Comment by minrk** [comment]
It works!

**Comment by minrk** [comment]
Two of three patches have been sent and submitted upstream, and can be removed for the next release. The last (boost-thread) I don't want to, since it only seems to affect conda-forge OS X builds (same recipe works fine on my own Macs). I don't know what the root of the issue is, but linking an extra boost component doesn't seem like a big deal.

File: recipes/mshr/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+  -DCMAKE_INCLUDE_PATH="$INCLUDE_PATH" \
+  -DCMAKE_LIBRARY_PATH="$LIBRARY_PATH" \
+  -DBOOST_ROOT="$PREFIX" \
+  -DENABLE_TESTS=1 \
+  -DMSHR_ENABLE_VTK=0 \
+  ..
+
+make -j${CPU_COUNT} VERBOSE=1
+make install
```
Comment (ocefpaf) [inline recipes/mshr/build.sh]: Is there a make check we can run here?

Comment (minrk) [inline recipes/mshr/build.sh]: Quite possibly. I'll check once the feedstock is up.

**Pull request merged**